samples/csharp_dotnetcore/06.using-cards/Startup.cs

```diff
@@ -20,6 +26,24 @@ public void ConfigureServices(IServiceCollection services)
             // Create the Bot Framework Adapter with error handling enabled.
             services.AddSingleton<IBotFrameworkHttpAdapter, AdapterWithErrorHandler>();
 
+            // The Dialog that will be run by the bot.
+            services.AddSingleton<RootDialog>();
+
+            // Create the bot as a transient. In this case the ASP Controller is expecting an IBot.
+            services.AddTransient<IBot, RichCardsBot<RootDialog>>();
+
+            // Components common to all adaptive dialogs.
+            ComponentRegistration.Add(new AdaptiveComponentRegistration());
```

Comment:
These component registrations are not needed.  The bot runs exactly the same without them.

Reply:
Removing..,